assets/queries/dockerfile/healthcheck_instruction/query.rego

```diff
@@ -0,0 +1,18 @@
+package Cx
+
+CxPolicy [ result ] {
+  resource := input.document[i].command
+  not contains(resource, "healthcheck")
+
+	result := {
+                "documentId": 		input.document[i].id,
+                "searchKey": 	    sprintf("%s", [resource[0].Cmd]),
+                "issueType":		"MissingAttribute",  #"MissingAttribute" / "RedundantAttribute"
+                "keyExpectedValue": "Dockerfile contains instruction 'HEALTHCHECK'",
+                "keyActualValue": 	"Dockerfile doesn't contain instruction 'HEALTHCHECK'"
```

Comment:
Try to specify the faulty command to help the user identify it

Reply:
Since it's 'HEALTHCHECK' Cmd that is missing can't specify

Reply:
Yes, that does make sense, my bad.